docs: Add new images to display new web ui; change quick-start docs pages to reflect images. (#1069)
y-scope/clp: docs/src/user-guide/quick-start/clp-json.md

* `<path...>` are paths to JSON log files or directories containing such files.
  * Each JSON log file should contain each log event as a

>     [separate JSON object](./index.md#clp-json), e.g., *not* as an array.

Reviewer: Fundamentally, I think this sentence is a bit confusing, so I think using "e.g." won't help. Regardless, we should address this in another PR.

Author: ok I'll make a note of that.

>     [separate JSON object](./index.md#clp-json), e.g., *not* as an array.

Reviewer: ```suggestion
    [separate JSON object](./index.md#clp-json), i.e., *not* as an array.
```

your browser (if you changed `webui.host` or `webui.port` in `etc/clp-config.yml`, use the new
values).

[Figure 3](#figure-3) shows the search page after running a query.

(figure-3)=
:::{card}

:::{image} clp-search-new-ui-json.png

+++

> **Figure 3**: CLP's web UI search page.

Author: no thank you

:::

The numbered circles in [Figure 3](#figure-3) correspond to the following features:

1. Enter a query in the search box. The format of your query should conform to CLP's
   [JSON search syntax](../reference-json-search-syntax.md).
2. Toggle case sensitivity by clicking here.
3. Filter for logs in a specific timeframe by setting the start time and end time, or you can
   select a preset filter (i.e. `Last 15 minutes`; `Yesterday`) from the dropdown list. The default
   filter is `All time`.
4. After you submit a query, the timeline shows the number of results across the specified time
   range of your query.
   * You can click and drag to zoom into a time range.
   * When you mouse over a specific timeframe bar, a popup will display the range and the number of
   search results in that range.
5. The table below the timeline displays the search results for your query.
   * You can sort by `Ascending` or `Descending` timestamp by clicking the `Timestamp` header in the table.
   * Each search result will provide a link to the original file in which the log message is found.
6. Click the `Ingest` link to get statistics about your compression jobs.

> 

Author: we have decided to refrain from using the include directive at present.

your browser (if you changed `webui.host` or `webui.port` in `etc/clp-config.yml`, use the new
values).

[Figure 3](#figure-3) shows the search page after running a query.

(figure-3)=
:::{card}

:::{image} clp-search-new-ui-json-2.png

+++

> **Figure 3**: CLP's web UI search page.

Reviewer: ```suggestion
**Figure 3**: The search page in CLP's UI.
```

:::

> The numbered circles in [Figure 3](#figure-3) correspond to the following features:

Reviewer: ```suggestion
The numbered circles in [Figure 3](#figure-3) correspond to the following elements:
```

> 1. **The query input box**. The format of your query should conform to
>    CLP's [JSON search syntax](../reference-json-search-syntax.md).

Reviewer: ```suggestion
1. **The query input box**. The format of your query should conform to CLP's
   [JSON search syntax](../reference-json-search-syntax.md).
```

Sorry, I wrapped this incorrectly in my previous suggestion.

> 1. Enter a query in the search box. The format of your query should conform to CLP's
>    [JSON search syntax](../reference-json-search-syntax.md).
> 2. Toggle case sensitivity by clicking here.
> 3. Filter for logs in a specific timeframe by setting the start time and end time, or you can
>    select a preset filter (e.g. `Last 15 minutes`; `Yesterday`) from the dropdown list. The default
>    filter is `All time`.
> 4. If you've compressed multiple different datasets, and you'd like to confine your search to a
>    specific set, select it from the drop-down menu.
> 5. After you submit a query, the timeline shows the number of results across the specified time
>    range of your query.
>    * You can click and drag to zoom into a time range.
>    * When you mouse over a specific timeframe bar, a popup will display the range and the number of
>    search results in that range.
> 6. The table below the timeline displays the search results for your query.
>    * You can sort by `Ascending` or `Descending` timestamp by clicking the `Timestamp` header in the table.
>    * Each search result will provide a link to the original file in which the log message is found.
> 7. Click the `Ingest` link to get statistics about your compression jobs.

Reviewer: ```suggestion
1. **The query input box**. The format of your query should conform to
   CLP's [JSON search syntax](../reference-json-search-syntax.md).
2. **The query case-sensitivity toggle**. When turned on, CLP will search for log events that match
   the case of your query.
3. **The time range selector**. CLP will search for log events that are in the specified time range.
   You can select a preset filter (e.g., `Last 15 minutes`; `Yesterday`) from the dropdown, or
   choose `Custom` and set the start time and end time directly.
4. **The dataset selector**. CLP will search for log events that belong to the selected dataset.
5. **The search results timeline**. After a query, the timeline will show the number of results
   across the time range of your query.
   * You can click and drag to zoom into a time range.
   * When you mouse over a bar in the timeline, a popup will display the range and the number of
     search results in that range.
6. **The search results**.
   * You can sort by `Ascending` or `Descending` timestamp by clicking the `Timestamp` header in the
     table.
   * Each search result includes a link to the original file in which the log event was found.
```

> 7. Click the `Ingest` link to get statistics about your compression jobs.

Reviewer: I suggest removing this (and the annotation in the image) since it seems a bit out of place. I think users should be able to navigate to the page and figure out what it does without us explaining it. Sidenote: I dislike that it's called "Ingest", but that's a larger discussion to be had with the cloud team.

Author: @kirkrodrigues removing the whole thing? your comment was applied to all seven points in the list

Reviewer: Just point 7.
